spec/requests/swagger_docs/v6/full_assessment_spec.rb

```diff
@@ -100,8 +100,7 @@
                             },
                             employed: {
                               type: :boolean,
-                              example: true,
-                              description: "Applicant's partner is employed",
+                              description: "Deprecated - employment is always determined by presence of employment or self-employment records",
```

Comment:
'Presence of employment records' was the old method, I believe? How about this:
```suggestion
                              description: "Deprecated - employment is determined by presence of gross employment income",
```